src/custom/theme/cowSwapTheme.tsx

```diff
@@ -18,65 +21,94 @@ export function colors(darkMode: boolean): Colors {
+    white: '#FFFFFF',
+    black: '#000000',
 
     // ****** text ******
-    // text1: darkMode ? '#FFFFFF' : '#000000',
-    text2: darkMode ? '#DCDCDC' : '#565A69',
-    // text3: darkMode ? '#6C7284' : '#888D9B',
-    // text4: darkMode ? '#565A69' : '#C3C5CB',
-    // text5: darkMode ? '#2C2F36' : '#EDEEF2',
+    text1: darkMode ? '#000000' : '#000000',
+    text2: darkMode ? '#000000' : '#000000',
 
     // ****** backgrounds / greys ******
-    bg1: darkMode ? '#b0e1de' : '#b0e1de',
-    bg2: darkMode ? '#2C2D3F' : '#F7F8FA',
-    bg3: darkMode ? '#1E1F2C' : '#EDEEF2',
-    // bg4: darkMode ? '#565A69' : '#CED0D9',
-    // bg5: darkMode ? '#6C7284' : '#888D9B',
+    bg1: darkMode ? '#D5E9F0' : '#D5E9F0',
+    bg2: darkMode ? '#000000' : '#000000',
+    bg3: darkMode ? '#d5e8f0' : '#d5e8f0',
+    bg4: darkMode ? '#d5e8f0' : '#ffffff',
 
     // ****** specialty colors ******
-    // modalBG: darkMode ? 'rgba(0,0,0,.425)' : 'rgba(0,0,0,0.3)',
-    advancedBG: darkMode ? '#2B2D3F' : 'rgb(247 248 250)',
+    advancedBG: darkMode ? '#000000' : '#000000',
 
     // ****** primary colors ******
-    primary1: darkMode ? '#3F77FF' : '#8958FF',
-    // primary2: darkMode ? '#3680E7' : '#FF8CC3',
-    // primary3: darkMode ? '#4D8FEA' : '#FF99C9',
-    // primary4: darkMode ? '#376bad70' : '#F6DDE8',
-    primary5: darkMode ? '#153d6f70' : 'rgba(137,88,255,0.6)',
+    primary1: darkMode ? '#FF784A' : '#FF784A',
+    primary5: darkMode ? '#FF784A' : '#FF784A',
 
     // ****** color text ******
-    primaryText1: darkMode ? '#6da8ff' : '#8958FF',
+    primaryText1: darkMode ? '#000000' : '#000000',
 
     // ****** secondary colors ******
     secondary1: darkMode ? '#2172E5' : '#8958FF',
-    // secondary2: darkMode ? '#17000b26' : '#F6DDE8',
     secondary3: darkMode ? '#17000b26' : 'rgba(137,88,255,0.6)',
 
     // ****** other ******
-    // red1: '#FF6871',
-    // red2: '#F82D3A',
-    // green1: '#27AE60',
-    // yellow1: '#FFE270',
-    // yellow2: '#F3841E',
     blue1: '#3F77FF',
     purple: '#8958FF',
-    border: darkMode ? '#3a3b5a' : 'rgb(58 59 90 / 10%)',
+    border: darkMode ? '#000000' : '#000000',
     disabled: darkMode ? '#31323e' : 'rgb(237, 238, 242)'
   }
 }
 
+function themeVariables(colorsTheme: Colors) {
+  return {
+    logo: { src: Logo, alt: 'CowSwap Logo', width: '208px', height: '50px' },
+    body: {
+      background: css`
+        background: url(${cowBg}) no-repeat 100% / cover;
+      `
+    },
+    appBody: {
+      maxWidth: '440px',
+      boxShadow: `6px 6px 0px ${colorsTheme.black}`,
+      borderRadius: '8px',
+      border: `4px solid ${colorsTheme.black}`,
+      padding: '12px 6px 24px'
+    },
+    header: {
+      border: 'none'
+    },
+    swap: {
+      headerSize: '28px'
+    },
+    buttonLight: {
+      fontSize: '26px',
+      fontWeight: 'bold',
+      border: `4px solid ${colorsTheme.black}`,
+      borderRadius: '9px',
+      boxShadow: `4px 4px 0px ${colorsTheme.black}`
```

Comment:
why are we using px again? let's stick to `rem` unless absolutely necessary

Reply:
Existing codebase uses px's so just mimicking what was already in place.

Reply:
ok sgtm